shell/packages/sandstorm-db/profile.js

```diff
@@ -70,6 +70,26 @@ if (Meteor.isServer) {
 
   var Url = Npm.require("url");
   httpProtocol = Url.parse(process.env.ROOT_URL).protocol;
+
+  var Crypto = Npm.require("crypto");
+  Meteor.methods({
+    revealIdentity: function (token, identityId) {
```

Comment:
As explained in my comment in grainview.js, I don't think this should be a Meteor method.